default.nix

```diff
@@ -15,7 +15,7 @@ python3.pkgs.buildPythonApplication rec {
 
     # needed for interactive unittests
     python3.pkgs.pytest
-    nix_2_4
+    nixVersions.nix_2_4
```

Comment:
Does this work in 21.11 as well?

Reply:
I don't think so but I also don't know how to make it work in both versions.

Reply:
You could use `pkgs.nixVersions.nix_2_4 or nix_2_4`.

Reply:
That does not work because nixVersions is also undefined.

Reply:
the `or` syntax should work even for nested attribute sets.

Reply:
Now it works. I tried that before and I am sure that it didn't work.